tests/override_config_table/utilities.py

```diff
@@ -5,6 +5,8 @@
 
 logger = logging.getLogger(__name__)
 
+NON_USER_CONFIG_TABLES = ["FLEX_COUNTER_TABLE", "ASIC_SENSORS"]
```

Comment:
Do we have yang models for these 2 tables?

Reply:
We have FLEX_COUNTER_TABLE. I don't see yang model for ASIC_SENSORS

Reply:
Is it possible to introduce yang validation error?

Reply:
I think so. I will discuss with you and introduce yang validation in next PR.